diff -up tcp_wrappers_7.6-ipv6.4/Makefile.22~ tcp_wrappers_7.6-ipv6.4/Makefile
--- tcp_wrappers_7.6-ipv6.4/Makefile.22~	2024-01-11 12:10:54.777934788 +0100
+++ tcp_wrappers_7.6-ipv6.4/Makefile	2024-01-11 12:11:17.170160405 +0100
@@ -1,5 +1,7 @@
 # @(#) Makefile 1.23 97/03/21 19:27:20
 
+CC ?= gcc
+
 what:
 	@echo
 	@echo "Usage: edit the REAL_DAEMON_DIR definition in the Makefile then:"
@@ -740,7 +742,7 @@ $(LIB):	$(LIB_OBJ)
 	-$(RANLIB) $(LIB)
 
 $(SHLIB): $(LIB_OBJ)
-	gcc -shared -fPIC -Wl,-soname -Wl,$(SHLIB).$(MAJOR) \
+	$(CC) -shared -fPIC -Wl,-soname -Wl,$(SHLIB).$(MAJOR) \
 		-o $(SHLIB).$(MAJOR).$(MINOR).$(REL) $^ $(LDFLAGS) $(LIBS)
 	ln -s $(SHLIB).$(MAJOR).$(MINOR).$(REL) $(SHLIB).$(MAJOR)
 	ln -s $(SHLIB).$(MAJOR).$(MINOR).$(REL) $(SHLIB)
